perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -1094,7 +1094,9 @@ sub waste_munge_report_data {
     my $address = $c->stash->{property}->{address};
     my $service = $c->stash->{services}{$id}{service_name};
     my $uprn = $c->stash->{services}{$id}{uprn};
-    $data->{title} = "Report missed $service";
+    my $service_description = _containers->{$c->stash->{services}{$id}->{service_id}}->{description};
```

Comment:
I think this needs to be $self->_containers and property needs to be passed in, otherwise it might have the wrong information for communal properties